src/sympc/tensor/grads/grad_functions.py

```diff
@@ -432,6 +435,160 @@ def backward(ctx: Dict[str, Any], grad: MPCTensor) -> MPCTensor:
         return grad.reshape(shape)
 
 
+class GradConv2d(GradFunc):
+    """The multiplication gradient function."""
+
+    @staticmethod
+    def function_aux(
```

Comment:
Q: Could we rename this? Maybe to something like `get_grad_input_padding`?

Reply:
Could we also add a test (in the `grad_test` py file) only for this method?

Reply:
sure :D